module.yml

```diff
@@ -71,8 +75,8 @@ Parameters:
     Description: 'Target read capacity utilization (in percent) that auto scaling tries to achieve (if you have spiky reads, a lower number is better; ignored when BillingAndScalingMode := ONDEMAND).'
     Type: Number
     Default: 80
-  Encryption: # TODO DynamoDB does not support customer managed CMKs. How should we handle this? Remove Encryption parameter and wait for customer managed CMK support using the kms-key module?
-    Description: 'Enable server side encryption using KMS (AWS managed) CMK.'
+  Encryption:
+    Description: 'Deprecated in v1, will be removed in v3, use KmsKeyModule instead! Enable server side encryption using KMS (AWS managed) CMK.'
```

Comment:
How can we remove that parameter without breaking backwards compatibility?

Reply:
I'm not sure if I get your question. We have to remove the parameter at some point and this will break the interface? But, in the meantime , we keep the parameter and things will work as before?

Reply:
I was asking about how we remove the parameter in `v3` without corrupting/deleting the DynamoDB table during an update.

Reply:
according to the cfn docs this should't be a replacement. the old items will still be encrypted with the aws/dynamodb default key.